contracts/evmscript/EVMScriptRegistry.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.4.18;
+pragma solidity ^0.4.18;
```

Comment:
I think this should be kept pinned; it's a pretty complicated contract.

`EVMScriptRunner` was left unpinned as it's subclassed by `AragonApp`.